homework_2/css/style.css

```diff
@@ -0,0 +1,166 @@
+a {
+	text-decoration: none;
+}
+
+.clearfix:after {
+  content: ""; /* добавить содержимое: "" */
+  display: block;  /* сделать блоком, т.к. inline не может иметь clear */
+  clear: both;  /* с обеих сторон clear */
+}
+
+body {
+	background-color: #f9f9f9;
+}
+
+/*HEADER*/
+
+h1 {
+	margin-left: 47px;
+	margin-bottom: 40px;
+	font-weight: bold;
+	font-size: 40px;
+	font-family: Verdana;
+}
+
+.head_link {
+	width: 96px;
+	padding: 10px 10px 10px 24px;
+	display: block;
+	color: white;
+	transition: background-color 0.5s;
+}
+
+.head_link:hover {
+	background-color: #363636;
+	border-left: 1px solid #838383;
+	border-right: 1px solid #838383;
+	padding: 10px 9px 10px 23px;
+}
+
+.container {
+	width: 1060px;
+	margin: 0 auto;
+	font-family: Time new roman;
+	background-color: white;
+}
+
+.head_menu {
+	/*display: inline-block; */
```

Comment:
don`t save comments with code in final version, you may create another branch for saving this comments or another version of implementation